--- chromium-84.0.4147.89.orig/third_party/libvpx/BUILD.gn
+++ chromium-84.0.4147.89/third_party/libvpx/BUILD.gn
@@ -411,6 +411,8 @@ static_library("libvp9rc") {
     } else {
       sources = libvpx_srcs_arm64
     }
+  } else if (current_cpu == "ppc64") {
+    sources = libvpx_srcs_ppc64
   }
   sources += [ "//third_party/libvpx/source/libvpx/vp9/ratectrl_rtc.cc" ]
   sources += [ "//third_party/libvpx/source/libvpx/vp9/ratectrl_rtc.h" ]
